tmail-backend/tmail-third-party/openpaas/pom.xml

```
@@ -67,6 +72,11 @@
             <artifactId>mockserver-netty</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.linagora.tmail</groupId>
```

```suggestion
            <groupId>${project.groupId}</groupId>
```

And should be placed on top, before the james dependencies